.github/workflows/release.yml

```
@@ -158,9 +166,9 @@ jobs:
     uses: ./.github/workflows/reusable_deploy_docs.yml
     with:
       CONCURRENCY: ${{ github.ref_name }}
-      PY_DOCS_VERSION_NAME: ${{ inputs.release-type == 'prerelease' && 'prerelease' || needs.version.outputs.final }}
-      RS_DOCS_VERSION_NAME: ${{ inputs.release-type == 'prerelease' && 'prerelease' || 'head' }}
-      UPDATE_LATEST: ${{ inputs.release-type == 'release' }}
+      PY_DOCS_VERSION_NAME: ${{ inputs.release-type != 'final' && 'prerelease' || needs.version.outputs.final }}
+      RS_DOCS_VERSION_NAME: ${{ inputs.release-type != 'final' && 'prerelease' || 'head' }}
```

swap the if-and else to avoid unnecessary negations

```suggestion
      PY_DOCS_VERSION_NAME: ${{ inputs.release-type == 'final' && needs.version.outputs.final || 'prerelease' }}
      RS_DOCS_VERSION_NAME: ${{ inputs.release-type == 'final' && 'head' || 'prerelease' }}
```